### ISSUE TITLE
Restore ability to scan tickets & view attendees in mobile apps without `ee_read_payment_methods` & `ee_read_others_payment_methods`

### ISSUE THREAD

**Issue by joshfeck** [eventespresso/event-espresso-core#903]
**Describe the bug**

Since the release of 4.9.74.p, mobile app users will need a WP user account with `ee_read_payment_methods` & `ee_read_others_payment_methods` capabilities to scan tickets and view attendee lists
**To Reproduce**
1. Set up a user account with Events Administrator role. Unless that role has been customized on your site, the user account will not have `ee_read_payment_methods` & `ee_read_others_payment_methods` caps
2. connect with the mobile app using the Events Admin account and try viewing a registration list or scan a ticket

**Expected behavior**

The user account should only need:
```
    ee_edit_checkin
    ee_edit_checkins
    ee_read_checkin
    ee_read_checkins
    ee_read_contact
    ee_read_contacts
    ee_read_events
    ee_read_private_events
    ee_read_registration
    ee_read_registrations
    ee_read_transaction
    ee_read_transactions
    ee_read_others_registrations
    ee_read_others_events
    ee_read_others_checkins
    ee_edit_others_checkins
```

**Screenshots**
![image from ios](https://user-images.githubusercontent.com/891156/51410596-20dac100-1b33-11e9-9888-05ee0170b157.png)

**Environment:**
 - Version of Event Espresso:  current master, back to 4.9.74.p (4.9.73.p is OK)

**Additional context**
related to #804

**Comment by mnelson4** [comment]
I have reproduced this on a site running EE 4.9.76, but not with 4.9.73.
Here's the query params on the now-failing requests:
```
{
    "where": {
        "Event.EVT_ID": "9",
        "Ticket.Datetime.DTT_ID": "1",
        "STS_ID": [
            "!%3D",
            "RIC"
        ]
    },
    "group_by": [
        "REG_ID"
    ],
    "limit": "0,20",
    "order_by": {
        "Attendee.ATT_full_name": "ASC"
    },
    "include": "Attendee.*,Ticket.*,Transaction.Payment_Method.PMD_name,Status.STS_code,Event.EVT_ID,Ticket.Datetime.*",
    "calculate": "datetime_checkin_stati",
    "default_where_conditions": "full_this_minimum_others",
    "caps": "read_admin"
}
```
Notice `"include":"...Transaction.Payment_Method.PMD_name..."`. And because it has `"caps":"read_ADMIN"`, it's technically correct that they should have `ee_read_payment_methods` and `ee_read_other_payment_methods`.

So it seems there was a "bug" that was "fixed" in 4.9.73... that caused this :disappointed: ...

It seems the old behaviour was that the innaccessible field, "payment_method", was simply null-ed out on the response, whereas now it's replacing the entire response with an error.

**Comment by mnelson4** [comment]
Fixed by #904
